src/gluonts/torch/model/deepar/module.py

```diff
@@ -132,6 +132,7 @@ def __init__(
             else [min(50, (cat + 1) // 2) for cat in cardinality]
         )
         self.lags_seq = lags_seq or get_lags_for_frequency(freq_str=freq)
+        self.lags_seq = [l - 1 for l in self.lags_seq]
```

Comment:
Maybe I'm completely not getting the issue here, but if we do `l-1` for the lags, don't we feed in the current time stamp during training when `l=1` thereby introducing a data leak? Maybe I'm wrong, though.

Reply:
It depends on what time index the lag is applied: for how the RNN input preparation is written, the lag index gets subtracted from `t`, when we have data up to time `t` and want a prediction for time `t+1`. The `test_rnn_input` test function aims at verifying precisely this. Let me try to see whether the test can be made more apparent.